scripts/serverless.js

```diff
@@ -46,8 +51,29 @@ process.once('uncaughtException', (error) =>
     serverless,
     hasTelemetryBeenReported,
     commandUsage,
-  })
-);
+  });
+  clearTimeout(keepAliveTimer);
+});
+
+process.on('SIGINT', () => {
+  if (commandSchema) {
```

Comment:
I believe we also need to check `hasTelemetryBeenReported`, `isTelemetryDisabled` and `(serverless ? serverless.isTelemetryReportedExternally : true)`

Reply:
Of course - I've confirmed with all other places where we have these guards that above lists all needed checks :+1: